app/controllers/flows_controller.rb

```diff
@@ -680,10 +718,12 @@ def generate_state_file_intake(params)
       _type = params.keys.find { |k| k.start_with?('submit_') }&.sub('submit_', '')&.to_sym
       attributes = self.class.send("#{@us_state}_attributes",
         first_name: params[:flows_controller_sample_intake_form][:first_name],
-        last_name: params[:flows_controller_sample_intake_form][:last_name]
+        last_name: params[:flows_controller_sample_intake_form][:last_name],
+                                   filing_status: _type
       )
       intake_class = StateFile::StateInformationService.intake_class(@us_state)
       intake = intake_class.create(attributes)
+      intake.efile_submissions.create!
```

Comment:
wow this seems pretty significant. it's a little strange to have a submission from the beginning but i guess the flow explorer fills in a lot a the beginning. was it causing an error to not have a submission?

Reply:
@jenny-heath yeah it wasn't creating a submission on its own for some reason but not sure if this is the best way to go about it

Reply:
maybe the intention was for it to fill out up to the point before hitting submit and you're supposed to click that button manually?

Reply:
@jenny-heath yes! that was the previous way it worked should i keep as that? that makes sense to me